### ISSUE TITLE
Mod Inclusion Project: Craftable Gun Pack

### ISSUE THREAD

**Issue by DracoGriffin** [CleverRaven/Cataclysm-DDA#24966]
Use the following issue as a space to discuss the inclusion of:

Mod Ident: craftgp
Name: Craftable Gun Pack

Your comment should include:

### Suggestions for induction
(what parts of the mod should be included, can be as simple as just an item, a location or the complete inclusion)

### Incompatibilities
(what parts of the mod should not be included, either some objects / items or entire mod is conflicting with DDA theme [ for whatever reason ])

**Comment by DracoGriffin** [comment]
I normally reserve my comments/review to reduce swaying others' opinions and viewpoints, but I felt this one was especially important to make an initial point.

Gunpowder has already been [specifically removed](https://github.com/CleverRaven/Cataclysm-DDA/pull/12655) and [discussed previously](https://github.com/CleverRaven/Cataclysm-DDA/issues/21962), so this issue isn't the place for that. With that said, I do believe
>[If there are locations where gunpowder would reasonably appear in bulk, feel free to call them out](https://github.com/CleverRaven/Cataclysm-DDA/pull/12655#issuecomment-111742032)

is prudent for potentially looking at all the new locations that have been added (and continuing), that may benefit from this. Additionally, with the new player-faction base, it could be quite plausible as a relations-trade with the faction Old Guard that you might send them a caravan of ingredients in return for some supplies. Whether this is added in as an end-"quest" at the Commune/Refugee Center with the Old Guard or a trading "job" for one of your NPCs (with risks of loss from bandits and the sort). Likewise, there could potentially even be factory-type locations in some locations that could also have something similar to this.

In short, I believe there are ideas that can be expanded on this, but respect that player-crafting modern gunpowder is not an option.

**Comment by FulcrumA** [comment]
I like the idea of craftable guns, but not necessarily the way they work in the mod, where some recipes are kinda unbalanced or suspicious in general. 

I certainly would like current functionality of guns we already can make in games expanded with ability to choose what caliber they'd be upon decision to craft, though. Bullets already carry some stats with them, so - likely would need testing - the balance wouldn't be impacted much, especially with some better rounds already being balanced as well in terms of availability (and availability of actual regular guns able to fire them).

**Comment by DracoGriffin** [comment]
>As the whole point of the mod is to circumvent the prohibition on both craftable gunpowder and craftable firearms in the main game, I'm unclear what's left.

Just wanting to explore potential aspects of it as outlined above, but with the response as it is, I believe this is sufficient enough to close and move onto the next mod.
